src/js/edu-benefits/containers/BenefitsSelection.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+
+import BenefitsSelectionFields from '../components/BenefitsSelectionFields';
+import NavButtons from '../components/NavButtons';
+import { isValidSection } from '../utils/validations';
+import { withRouter } from 'react-router';
+import { veteranUpdateField, ensureFieldsInitialized, updateCompletedStatus } from '../actions/index';
+
+class BenefitsSelection extends React.Component {
+  render() {
+    const { section, currentLocation, data, submission, router, onStateChange, dirtyFields, setComplete } = this.props;
+    const navigateTo = path => router.push(path);
+
+    return (
+      <div className="form-panel">
+        <BenefitsSelectionFields data={data} section={section} onStateChange={onStateChange}/>
```

Comment:
The only thing to keep in mind is that the styling of the questions on the review page is different from the section itself, so we won't be able to use this component on the review page as is. We can leave this as is for now, but would love to brainstorm with you more on the best way to create reusable components that we can use both on the original section page and the review page.